src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/TransformGenerators.cs

```diff
@@ -330,4 +330,26 @@ public override string GenerateTransformer()
         }
     }
 
+    internal class ImageLoading : TransformGeneratorBase
+    {
+        public ImageLoading(PipelineNode node) : base(node)
+        {
+        }
+
+        internal override string MethodName => "LoadRawImageBytes";
+
+        public override string GenerateTransformer()
+        {
+            string inputColumn = InputColumns.Count() == 1 ? InputColumns[0] : throw new Exception($"output columns for the suggested transform: {MethodName} are null");
+            string outputColumn = OutputColumns.Count() == 1 ? OutputColumns[0] : throw new Exception($"output columns for the suggested transform: {MethodName} are null");
+            string inputFolder = Properties["imageFolder"] as string;
+            if (inputFolder == null)
+            {
+                throw new Exception($"miss imageFolder property");
```

Comment:
And in modelBuilder, we don't require ImageFolder to be set (we use full path), so maybe instead of throwing an error, giving a hint msg might be better choice?

Reply:
good idea I will remove it. I prefer full paths any day

---
In reply to: [339325224](https://github.com/dotnet/machinelearning/pull/4395#discussion_r339325224) [](ancestors = 339325224)